analysis/Scripts/Efficiencies/Selection_eff.py

```diff
@@ -0,0 +1,80 @@
+import ROOT, os
+from ROOT import TChain, TCanvas, TH1
+
+#directory = "/data/bfys/jdevries/gangadir/workspace/jdevries/LocalXML/"
+#directory = "/dcache/bfys/jdevries/ntuples/LcAnalysis/ganga/"
+#job = "78"
+ID = ""
+
+ybins = [[2.0, 2.5], [2.5, 3.0], [3.0, 3.5], [3.5, 4.0]]
+ptbins = [[3000, 4000], [4000, 5000], [5000, 6000], [6000, 8000], [8000, 20000]]
+
+#cuts = "lcplus_P < 300000 && lcplus_OWNPV_CHI2 < 80 && pplus_ProbNNp > 0.5 && kminus_ProbNNk > 0.4 && piplus_ProbNNpi > 0.5 && pplus_P < 120000 && kminus_P < 115000 && piplus_P < 80000 && pplus_PIDp > 0 && kminus_PIDK > 0"
+cuts = "lcplus_P < 300000 && lcplus_OWNPV_CHI2 < 80 &&  pplus_P < 120000 && kminus_P < 115000 && piplus_P < 80000"
```

Comment:
Ditto, load from imports

Reply:
I'm working on changing these scripts more fundamentally for inclusion with snakemake, these are just the unmodified scripts from Simon. So I can make these changes on these scripts, but they (the scripts) will be obsolete quite soon and the changes your talking about will be included in the new file anyways